### ISSUE TITLE
Wrong class on "week header" when timeline is selected

### ISSUE THREAD

**Issue by itajackass** [fullcalendar/fullcalendar#7762]
### Reduced Test Case

https://codepen.io/itajackass/pen/KKjLGRq

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

No connector (vanilla JS)

### Bug Description

Hi, I thing there is a problem with class assigned to "header table week" See my screenshot here:

With firebug you can see that "fc-sun" is assigned to header week cell. No sense for me. 

(problem has the same logic for year timeline view... with month row header with classes "fc-day fc-day-{NAMEOFDAY}")

Also is a good implement to assign a class "fc-week fc-week-{NUMBER}" to this table row to differentiate it from days row.

![immagine](https://github.com/user-attachments/assets/e655b465-7ee2-4cf1-84d6-13bc00a92ec6)

**Comment by acerix** [comment]
I don't see those classes in your demo, it looks like this when I inspect:

![Screenshot_2024-09-17_21-14-26](https://github.com/user-attachments/assets/9664d673-dcc5-4beb-8691-c09950aef247)

But I tested a basic example and can see the class names like `fc-day-sun` are applied to multi-day labels, so that does seems like a bug:

https://codepen.io/acerix/pen/jOjjprG?editors=0110

![Screenshot_2024-09-17_21-27-18](https://github.com/user-attachments/assets/77d702a9-f02f-4c4a-9a9c-919f98cbee22)

**Comment by itajackass** [comment]
> I don't see those classes in your demo, it looks like this when I inspect:
> 

yes because you are into "day" view... if you switch into week or month view (or year if available) you can see the class bug. (as you found then with our example)

**Comment by itajackass** [comment]
> ![Screenshot_2024-09-17_21-14-26](https://private-user-images.githubusercontent.com/379534/368387970-9664d673-dcc5-4beb-8691-c09950aef247.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY2NDE5OTAsIm5iZiI6MTcyNjY0MTY5MCwicGF0aCI6Ii8zNzk1MzQvMzY4Mzg3OTcwLTk2NjRkNjczLWRjYzUtNGJlYi04NjkxLWMwOTk1MGFlZjI0Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxOFQwNjQxMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZGNlNDhjMjRkOGYxZmQ2ZWNhYzE1ZTJkMDM4NGRkOGVjM2MzNzg1YzhmOWJhOTgxMzQzNzJkZmE4ZWJjYjY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2lg20Btvm8GOhTImTHukLsYEKnyvFnoPp1y-9qswLZA)

Also: is it possible to add a class to differentiate the headers row for weeks and months?

So in this screeshot convert from (in day view):

    <th class="fc-widget-header" data-date="2024-09-04T00:00:00" colspan="48">

to:

    <th class="fc-widget-header **fc-header-week fc-header-week-36**" data-date="2024-09-04T00:00:00" colspan="48">


And when in week,month,year view from:

    <th class="fc-widget-header fc-sun fc-past" data-date="2024-09-01" colspan="7"><div class="fc-cell-content" style="...."><span class="fc-cell-text fc-sticky" data-sticky-center="" style="....">W 36</span></div></th>

to:

    <th class="fc-widget-header **fc-header-week fc-header-week-36** fc-past" data-date="2024-09-01" colspan="7"><div class="fc-cell-content" style="...."><span class="fc-cell-text fc-sticky" data-sticky-center="" style="....">W 36</span></div></th>
